src/Stats.cc

```diff
@@ -175,11 +175,12 @@ void ProfileLogger::Log()
 			stats.nfa_states, stats.dfa_states, stats.computed, stats.mem / 1024));
 		}
 
+	// TODO: this also used to include a call to cq_memory_allocation from the CQ timer manager.
```

Comment:
We can just remove the "mem" part for now. Hardly anybody pays attention to this output anyways, and that way it's at least not confusing if somebody does.